tests/schema/tests.py

```diff
@@ -1914,6 +1914,28 @@ def test_add_foreign_key_long_names(self):
         with connection.schema_editor() as editor:
             editor.add_field(BookWithLongName, new_field)
 
+    @isolate_apps('schema')
+    @skipUnlessDBFeature('supports_foreign_keys')
+    def test_add_foreign_key_quoted_db_table(self):
+        class Author(Model):
+            class Meta:
+                db_table = '"table_author_double_quoted"'
+                app_label = 'schema'
+
+        class Book(Model):
+            author = ForeignKey(Author, CASCADE)
+
+            class Meta:
+                app_label = 'schema'
+
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+            editor.create_model(Book)
+        if connection.vendor == 'mysql':
```

Comment:
Is it possible to use something from `connection.ops` or similar to avoid a vendor check? Otherwise, looks good.

Reply:
I couldn't find anything useful. We don't have quotation mark in `connection.ops` :confused: .